components/Users/Actions/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import { Query, Mutation } from 'react-apollo'
+import gql from 'graphql-tag'
+
+import {
+  Loader,
+  InlineSpinner
+} from '@project-r/styleguide'
+
+import {
+  displayDate,
+  displayStyles,
+  displayDateTime,
+  Section,
+  SectionTitle,
+  DL,
+  DT,
+  DD
+} from '../../Display/utils'
+import DeleteUser from './DeleteUser';
+
+const GET_ACTIONS = gql`
+  query actions($id: String) {
+    user(slug: $id) {
```

Comment:
Can you rename `$id` to `$slug` to avoid future errors? IIRC we had a bug related to a similar naming issue recently.